lib/mysql.js

```diff
@@ -199,6 +199,55 @@ MySQL.prototype.executeSQL = function(sql, params, options, callback) {
     client.getConnection(executeWithConnection);
   }
 };
+ 
+/**
+ * Replace if the model instance exists with the same id or create a new instance
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model instance data
+ * @param {Function} [callback] The callback function
+ */
+
```

Comment:
@bajtos: Maybe it makes more sense to have an argument in `MySQL.prototype.updateOrCreate` for `replaceOrCreate` and if value `true` is passed, we get the fields by calling `this.buildReplaceFields(model, data);` as opposed to `var fields = this.buildFields(model, data);so no need for duplicate code here, thoughts? However that requires a few changes where you were calling `updateOrCreate`,...